### DISCUSSION TITLE
Web interface crashing after forced reload

### DISCUSSION THREAD

**Discussion by lucasbracher** [webpack/webpack#16013] [General]
Hello! I have an application running with webpack and I have the following problem:

I have a page whose URL is `/devices/1234`. When I refresh it, all the UI crashes. Even pressing `Alt+<-` to go to last page, it continues broken. I tried the solution proposed here -> https://stackoverflow.com/questions/64519310/ without success.
What I perceived is, when I do that, assets that are located on `/assets` are now looked on `/devices/assets`, generating lots of 404 errors . 

Any ideas on how can I investigate it?

Thanks in advance!

**Comment by vankop** [comment]
you probably need to set `publicPath`
